similar/main/net_udp.cpp

```diff
@@ -3820,7 +3820,7 @@ int net_udp_setup_game()
 		Netgame.Tracker = 0;
 #endif
 
-	return i >= 0;
+	return (i >= 0) ? window_event_result::close : window_event_result::handled;
```

Comment:
OK, yeah I used parentheses a lot, I thought it would give more clarity but I can leave them out in future.